picoquic_t/picoquic_t.c

```diff
@@ -170,16 +169,6 @@ static const picoquic_test_def_t test_table[] = {
     { "stress", stress_test },
     { "fuzz", fuzz_test },
     { "fuzz_initial", fuzz_initial_test},
-    { "h3zero_integer", h3zero_integer_test },
-    { "qpack_huffman", qpack_huffman_test },
-    { "h3zero_parse_qpack", h3zero_parse_qpack_test },
-    { "h3zero_prepare_qpack", h3zero_prepare_qpack_test },
-    { "h3zero_stream_test", h3zero_stream_test },
-    { "parse_demo_scenario", parse_demo_scenario_test },
-    { "h3zero_server", h3zero_server_test },
-    { "h09_server", h09_server_test },
-    { "generic_server", generic_server_test},
-    { "esni", esni_test }
```

Comment:
I understand why you are moving the ESNI test to httptest_t, since it is implemented on top of the basic http test. I will open an issue to move it back to the quic test library, because it really belongs there.